code/ui/QuickShop.scss

```diff
@@ -1,25 +1,24 @@
 QuickShop {
-    background-color: #0009;
-    border: 1px solid #000;
+    background-color: #000c;
+    border: 1px solid #1b1b1b;
     border-radius: 8px;
 
     font-family: Poppins;
-    max-width: 800px;
-    min-width: 600px;
+    width: 600px;
```

Comment:
I don't like the strict way. With a min-max, you are on the save side regarding rendering, border-sizes etc., as the best fitting size will be taken.

Reply:
Ok, I reverted this